erpnext/patches/v10_0/deduplicate_offline_pos.py

```diff
@@ -0,0 +1,26 @@
+import frappe
+
+
+def execute():
+    duplicates = frappe.db.sql(
+        'select name, offline_pos_name from `tabSales Invoice` where '
+        'offline_pos_name is not null group by '
+        'offline_pos_name having count(*) > 1'
+    )
+
+    for duplicate in duplicates:
+        keep, offline_pos_name = duplicate
+        duplicate_list = frappe.db.sql(
+            'select name from `tabSales Invoice` where '
```

Comment:
Use quote  """ """"
For reference check below link
https://github.com/frappe/erpnext/blob/develop/erpnext/patches/v10_0/update_user_image_in_employee.py#L10-L19